### ISSUE TITLE
Evaluation Model Performance

### ISSUE THREAD

**Issue by hosytuyen** [LukasStruppek/Plug-and-Play-Attacks#8]
Hi Lukas,

Thanks for sharing interesting work.

I am currently try to reproduce the result in the paper. I followed your code to train the Inception_V3 on CelebA for evaluation model. According to table 4 in the paper, the performance should be 93.28%, however, my trained model only got 88.25%.

I conduct my experiment with 4xA5000 with the training config: configs/training/evaluation/Inception_v3_CelebA.yaml

Understand that the performance of evaluation might affect the attack accuracy significantly. 
May I have your help in providing the pre-trained weights or any suggestion to improve training of the evaluation model?

Thank you very much.

**Comment by LukasStruppek** [comment]
Hi HoSyTuyen,

we already provide the pre-trained weights within the GitHub release, see [here](https://github.com/LukasStruppek/Plug-and-Play-Attacks/releases/tag/v1.0.0). The Inception-v3 models are the evaluation models. 

Regarding the drop in test accuracy for CelebA, it may be due to the fact that we used a custom cropped CelebA version with an improved image quality. Details on how to generate this dataset are stated in Appx. C in the paper. We also used weights pre-trained on FaceScrub as initialization to further improve the accuracy of the evaluation model, see Appx. B.2.

I hope this solves the problem. If you occur any other problems, I'm happy to help :)

Best,
Lukas
